unfoldingWord/translationCore
# Feature #2605/Reorder the result cards on the Door43 import search screen

#### This pull request addresses:

WIP: still working on testing.

Issue: #2605

SearchOptions - changed "Language Id" to "Language Code"
SearchResults - changed card order to title, user, language, book

#### How to test this pull request:

Click on Import from Door43 and should see the new arrangement.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3051)
